### DISCUSSION TITLE
What about a box64 UWP for Xbox One/Series consoles?

### DISCUSSION THREAD

**Discussion by danprice142** [ptitSeb/box64#283] [General]
Do you guys think its possible?

**Comment by ptitSeb** [comment]
But isn't those consoles already x86_64 PC, basicaly?

**Reply by toastmod** [reply]
I think the motivation is to use it as a userspace emulator. But box86/64 seem to be much more Linux oriented. A windows specific userspace emulator would be more appropriate for a UWP thing.
